src/toil/provisioners/aws/awsProvisioner.py

```diff
@@ -145,6 +145,20 @@ def _sshInstance(cls, leaderIP, *args):
         return ouput
 
     @classmethod
+    def rsyncLeader(cls, clusterName, args):
+        leader = cls._getLeader(clusterName)
+        logger.info("Rsync ready")
+
+        ssh_command = 'ssh -o "StrictHostKeyChecking=no"'  # Skip host key checking
```

Comment:
Add a ticket to remove this from  `ssh-cluster` and `rsync-cluster`. It's insecure. We cannot trade convenience for insecurity without giving the user a choice.

I'd be ok to add an `--insecure` argument to `ssh-cluster` and `rsync-cluster` that causes this flag to be added. That way the user can make a conscious choice about this.